homeassistant/components/switch/fritzhome.py

```diff
@@ -0,0 +1,66 @@
+"""
+Support for AVM Fritz!Box fritzhome switch devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/switch.fritzhome/
+"""
+import logging
+from homeassistant.components.fritzhome import DOMAIN
```

Comment:
See comment at the top how to import this domain.

Please add a blank line between import groups: standard python, 3rd party libraries and homeassistant imports.
https://www.python.org/dev/peps/pep-0008/#imports